modules/benchmark/R/align_data_to_data_pft.R

```diff
@@ -50,6 +50,7 @@
 #'  custom_table = table)
 #' }
 #' @export
+#' @importFrom rlang .data
```

Comment:
I don't see anywhere in this function that uses .data, so unless I missed one I don't think you need this `importFrom`